backend/core/src/listings/listings.service.ts

```diff
@@ -72,22 +114,29 @@ export class ListingsService {
       })
     }
 
-    if (jsonpath) {
-      listings = jp.query(listings, jsonpath)
+    // TODO(https://github.com/CityOfDetroit/bloom/issues/135): Decide whether to remove jsonpath
+    if (params.jsonpath) {
```

Comment:
@seanmalbert I think we would like to get rid of this too, but that requires adding a filter for listing.property.buildingAddress.county value.

Reply:
I noted this on our open issue to remove jsonpath:
https://github.com/CityOfDetroit/bloom/issues/135